src/dataset/DatasetTree.ts

```diff
@@ -144,7 +144,7 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
      * @returns {IZoweDatasetTreeNode | undefined} Returns matching profile node if found. Otherwise, returns undefined.
      */
     public findMatchingProfileInArray(datasetProvider: IZoweDatasetTreeNode[], profileName: string): IZoweDatasetTreeNode|undefined {
-        return datasetProvider.find((treeNode) => treeNode.label === profileName );
+        return datasetProvider.find((treeNode) => treeNode.label.includes(profileName) );
```

Comment:
@katelynienaber For `findMatchingProfileInArray()`, can we please keep `treeNode.label === profileName` instead of `includes()`?

Using `includes()` creates an issue where it will find the wrong node if the user favorites an item from a profile **abc1** before favoriting an item from profile **abc**. (In the gif below, the two profile names are "zoweCLI2" and "zoweCLI", respectively. Notice that items from "zoweCLI" are getting favorited under "zoweCLI2".):
![ze-rename-includes](https://user-images.githubusercontent.com/45975633/92699990-44600a00-f31c-11ea-83b8-b01235a25a7a.gif)

Reply:
I reverted this in the latest commit :)

Also allowed same name on different level of the tree, changed error message...